### PR TITLE
Fix issue where scroll bars on main window appear when log shows

### PR DESCRIPTION
**Pull request by dhensby**

### CODE REVIEW

File: sass/style.sass [hunk 1]
```diff
@@ -587,7 +587,8 @@ button.sidebar-open
 	max-height: 450px
 	background: #333
 	color: #fff
-	margin-top: 0
+        margin-top: 0
```
Comment (halkyon) [inline sass/style.sass]: Tabs not spaces :)

Comment (dhensby) [inline sass/style.sass]: bloody vim

**Comment by halkyon** [comment]
You'll also need to compile the CSS and commit the resulting file as well, since you're just changing an SCSS file here, which does nothing by itself :) `cd deploynaut && npm update && npm build`

**Comment by dhensby** [comment]
hmm - literally just did that and no changes...

**Comment by dhensby** [comment]
OK, so `npm install` goes and installs all the deps I need, but `npm update` and `npm build` do nothing :/

**Comment by dhensby** [comment]
meh, just did `npm install -g node-sass && node-sass -r ./sass/style.sass -o ./css/` and it worked

**Comment by dhensby** [comment]
Whoop - first contribution to 'naut. I'm practically a cloudbuster now!

**Comment by ss23** [comment]
Dammit @dhensby I knew there was a reason our cloud server went down, YOU BUSTED IT.

**Comment by dhensby** [comment]
Hot-diggity-dog! I knew that `margin-bottom` enabled a hack that allowed us to calculate the number of seconds to offset UTC by to ensure deploys started correctly and allowed us to prevent race conditions on deployments!!!

**Pull request merged**